crypto3/libs/blueprint/include/nil/blueprint/zkevm_bbf/small_field/opcodes/add_sub.hpp

```diff
@@ -0,0 +1,161 @@
+//---------------------------------------------------------------------------//
+// Copyright (c) 2024 Dmitrii Tabalin <d.tabalin@nil.foundation>
```

Comment:
Switch  on zkevm-s circuit for `blueprint_single_thread_zkevm_bbf_opcodes_add_sub_test` and  `blueprint_single_thread_zkevm_bbf_opcodes_cmp_test`. Now new opcodes in fact are not tested in CI.

Reply:
Done